### ISSUE TITLE
Color Picker Support IOS Devices ?

### ISSUE THREAD

**Issue by osamaraza24** [judah4/HSV-Color-Picker-Unity#26]
HSV Field uses Compute shader to display Gradient Color. But normally compute shader doesn't support on IOS Devices.

**Comment by judah4** [comment]
I think there is an override to not use the compute shader. Have you been able to test it on an IOS device?

**Comment by abhidec1** [comment]
it works fine on iOS devices. I simply drag and dropped the prefab into my scene. Tested on xsmax and iphone 6 plus.

**Comment by Segelzwerg** [comment]
We use it on a iPad with 12.4 and iOS 13. On both devices the gradient field is white. The same app works fine on multiple Android devices. Any advice?
Thanks a lot!

**Comment by Segelzwerg** [comment]
Turns out, I forgot to copy the Resource Folder from your Project to our.

**Comment by judah4** [comment]
Glad it's resolved!

**Comment by Segelzwerg** [comment]
I celebrated too early. The test I ran was with Unity Remote. The actual build does not work:
`2019-12-02 10:38:14.267949+0100 HourGlass[2581:110972] [DYMTLInitPlatform] platform initialization successful
2019-12-02 10:38:14.341499+0100 HourGlass[2581:110640] Built from '2019.2/release' branch, Version '2019.2.10f1 (923acd2d43aa)', Build type 'Release', Scripting Backend 'il2cpp'
2019-12-02 10:38:14.348803+0100 HourGlass[2581:110640] -> registered mono modules 0x1012708f0
-> applicationDidFinishLaunching()
2019-12-02 10:38:14.593814+0100 HourGlass[2581:110640] Metal GPU Frame Capture Enabled
-> applicationDidBecomeActive()
GfxDevice: creating device client; threaded=1
Initializing Metal device caps: Apple A7 GPU
Initialize engine version: 2019.2.10f1 (923acd2d43aa)
UnloadTime: 2.151125 ms
Metal: Compute shader (GenerateSVTexture.CSMain) has 1024 threads in a threadgroup which is more than the supported max of 512 in this device
Game started`

I tried to lower the lower the thread size as mentioned here: https://stackoverflow.com/questions/45460277/unity-metal-compute-shader-has-1024-threads-in-a-thread-group-which-is-more-tha/45647059#45647059, but it didn't help.
